Focusgroup/explainer.md

```diff
@@ -61,73 +60,93 @@ implementing the pattern:
-2. A set of focusable controls spanning a hierarchy of DOM can be added to a single logical 
-    focusgroup.
-3. A logical focusgroup can be configured to have wrap-around focus semantics if desired.
-4. A focusgroup can be configured to respond to either horizontal navigation keys or vertical
-    keys or both (to trivially reserve one axis of arrow key behavior for supplementary
-    actions, such as opening nodes in a tree view control).
-5. Focusgroups can be nested to provide logical navigation into multiple composed widgets (such
-    as lists within lists).
-6. Focusgroups can be used for grid-type navigation (structured content grids like &lt;table&gt;,
-    not CSS grids).
-
-## 6. Proposed Solution
-
-A new HTML attribute 'focusgroup'.
-
-The presence of this attribute on an element<sup><a href="#note1">1</a></sup> allows any of its
-focusable children (including those with `tabindex=-1`) to be focused using the keyboard arrow keys.
+1. (Child opt-in) Group a set of focusable child elements under a single focusgroup.
+2. (Descendent opt-in) Focusable elements deeply nested can participate in a single focusgroup.
+3. (Wrap) Focusgroup can be configured to have wrap-around focus semantics.
+4. (Horizontal/vertical) A focusgroup can be configured to respond to either horizontal navigation
+    keys or vertical keys or both (to trivially reserve one axis of arrow key behavior for 
+    supplementary actions, such as opening nodes in a tree view control).
+5. (Extend same direction) Focusgroups can be nested to provide arrow navigation into multiple 
+    composed widgets (such as lists within lists).
+6. (Extend opposite direction) Focusgroups can be nested to provide arrow navigation into composed
+    widgets with orthogonal navigation semantics (such as horizontal-inside-vertical menus).
+7. (Multiple focusgroups) Multiple focusgroups can be established on a single element (advanced CSS 
+    scenario).
+8. (Opt-out) Individual elements can opt-out of focusgroup participation (advanced CSS scenario)
+9. (Grid) Focusgroups can be used for grid-type navigation (structured content grids like &lt;table&gt;,
+    not "presentation" grids).
+
+## 6. Focusgroup Concepts
+
+A focusgroup is a group of elements that are related by arrow-key navigation and for which the web
+platform provides the arrow key navigation behavior by default (no JavaScript event handlers needed)!
+Navigation is provided according to order or structure of the DOM (not how the content is presented 
+in a user interface).
+
+This document describes two kinds of focusgroups: **linear focusgroups** and **grid focusgroups**.
+Linear focusgroups provide arrow key navigation among a *list* of elements. Grid focusgroups provide 
+arrow key navigation behavior for tabular (or 2-dimensional) data structures.
+
+Multiple linear focusgroups can be combined together into one logical focusgroup, but linear focusgroups
+cannot be combined with grid focusgroups and vice-versa.
```

Comment:
Maybe also specify that grid focusgroups can't be combined with other grid focusgroups?